opal/mca/common/ofi/common_ofi.c

```diff
@@ -554,11 +554,13 @@ static uint32_t get_package_rank(opal_process_info_t *process_info)
     }
 #endif
 
-    // Get the local peers
+    // Get the local peers - note there may not always be local peers,e.g. .singleton launch case
     OPAL_MODEX_RECV_VALUE(rc, PMIX_LOCAL_PEERS, &pname, &local_peers, PMIX_STRING);
     if (PMIX_SUCCESS != rc || NULL == local_peers) {
         // We can't find package_rank, fall back to procid
-        opal_show_help("help-common-ofi.txt", "package_rank failed", true);
+        if (10 <= opal_common_ofi_verbose_level) {
```

Comment:
This doesn't seem like the right fix.  We're now hiding a real issue in the non-singleton cases by not showing this error message.

Can't we add logic to detect that we shouldn't find data about our peers instead of just squelching a print?